src/screens/Gurbani/Navbar.tsx

```diff
@@ -0,0 +1,78 @@
+import { StackNavigationOptions } from '@react-navigation/stack'
+import React from 'react'
+import { Dimensions, Image, StyleSheet, View } from 'react-native'
+import Icon from 'react-native-vector-icons/Ionicons'
+
+import logo from '../../../assets/images/logo.png'
+import Colours from '../../themes/colours'
+import { px } from '../../themes/utils'
+import Typography from '../../components/Typography'
+
+const styles = StyleSheet.create( {
+  header: {
+    ...px( 10 ),
+  },
+  headerIcon: {
+    fontSize: 22,
+    color: Colours.White,
+    paddingLeft: 20,
+    paddingRight: 20,
+  },
+  headerRoot: {
+    position: 'absolute',
+    zIndex: -1,
+    width: Dimensions.get( 'screen' ).width,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  logo: {
+    width: 25,
+    height: 25,
+  },
+  root: {
+    backgroundColor: Colours.DarkGray,
+    flexDirection: 'row',
+    borderTopLeftRadius: 20,
+    borderTopRightRadius: 20,
+    width: Dimensions.get( 'window' ).width,
+    height: '100%',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    marginTop: 10,
+  },
+  rootContainer: {
+    backgroundColor: Colours.DarkestGray,
+    shadowColor: 'transparent',
+  },
+  statusBar: {
+    borderColor: 'red',
+    borderWidth: 2,
+    position: 'relative',
+    top: -20,
+  },
+} )
+
+/**
+ * Navbar component for main header.
+ */
+const Navbar = () => (
```

Comment:
This can also go in `src/components` ?

Reply:
I think we should only move to `src/components` once we're sure it's going to be used elsewhere or it's definitely a smaller block. I think it **probably** will be used elsewhere, but require tweaking before that's the case, so going to keep it with the screen until we do PRs for the other screens and find that this is the case. It could also be that screens just export a different type of navbar entirely, and this may not even be reusable.